pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -278,3 +280,10 @@ func setDefaultMiddleware(config *configuration.Configuration) {
 	}
 	return
 }
+
+func setDefaultLogParameters(config *configuration.Configuration) {
+	if len(config.Log.Fields) == 0 {
+		config.Log.Fields = make(map[string]interface{})
+	}
+	config.Log.Fields[audit.LogEntryType] = "message"
```

Comment:
What about `"generic"`? Can it be `const`?

~~If this meant only `docker/distribution` messages, I'd say `"distribution"` but since this covers our middleware messages as well, I'd call it *generic*. Any better ideas?~~

I'd prefer to use `openshift.logger` instead of `openshift.log.entry`.